addons/website/__init__.py

```diff
@@ -41,6 +41,11 @@ def rem_website_id_null(dbname):
 
 def post_init_hook(env):
     generate_primary_snippet_templates(env, 'website')
+    website_friend_modules = env['ir.module.module'].with_context(active_test=False).search([
+        ('name', 'like', r'website\_'),
+    ], order='name')
+    for website_friend_module in website_friend_modules:
+        generate_primary_snippet_templates(env, website_friend_module.name)
```

Comment:
Why do we need this?
Should it be the responsibility of the modules to do that in their post_init_hook if relevant in their module?
I'd not do it here in this generic way, what do you think?

Unless you actually meant to target theme modules, which indeed should not have to write any python (and so, nothing in their manifest)?

Then if we do that for themes, it might indeed makes sense to do it for website modules too, but this is not targeting themes right now

Reply:
Replaced it by a `<function>` record in the specific module data, which is way better as it allows to determine if it has to be run before or after specific records.